Deploying_vLLM/vllm_autocalc.py

```diff
@@ -22,17 +22,12 @@ def prepare_group_list(grouping_string):
     return group_list
 
 def vllm_auto_calc(fd):
-    if int(fd['VLLM_PROMPT_USE_FUSEDSDPA']) == 1:
-        gpu_free_mem_target = 1
-    else:
-        gpu_free_mem_target = 3
-
     # handle exceptions
     if (fd['MODEL'] in [
         'Qwen/Qwen2.5-32B-Instruct'
     ] and DTYPE == "fp8"
     and hpu_determined == "GAUDI2"):
-        gpu_free_mem_target = 3
+        fd['GPU_FREE_MEM_TARGET'] = 3
```

Comment:
We decided to keep GPU_FREE_MEM_TARGET for all Qwen models to 3 (in settings csv). Then this entire exception is not needed.